__tests__/interpolate/Interpolate_color_test.re

```diff
@@ -0,0 +1,39 @@
+open Jest;
+
+let it = test;
+
+describe("Interpolate_color", () => {
+  describe("lerpColorRGBA", () =>
+    it("should linearly interpolate between two colors", () => {
+      let a = Interpolate_color.{r: 0., g: 0., b: 0., a: 0.};
+
+      let b = Interpolate_color.{r: 100., g: 200., b: 255., a: 0.6};
+
+      Expect.(
+        expect(
+          Interpolate_color.lerpColorRGBA(~acc=a, ~target=b, ~roundness=0.5),
+        )
+        |> toEqual(Interpolate_color.{r: 50., g: 100., b: 127.5, a: 0.3})
+      );
+    })
+  );
+
+  describe("remapColor", () =>
+    it("should map an input numeric range to an output color domain", () => {
+      let a = Interpolate_color.{r: 0., g: 0., b: 0., a: 0.};
```

Comment:
To create a record in Reason its type definition needs to be brought into scope. That's why we locally open the `Interpolate_color` module here – it brings the `rgba` record type into scope so we can create an `rgba` record.